spec/lib/support/input_methods_spec.rb

```diff
@@ -5,11 +5,12 @@
     Class.new do
       include ActiveAdminAddons::InputMethods
 
-      attr_reader :method
+      attr_reader :method, :options
 
-      def initialize(object, method)
+      def initialize(object, method, options = {})
```

Comment:
Unused method argument - options. If it's necessary, use _ or _options as an argument name to indicate that it won't be used.